webapp/channels/src/components/more_direct_channels/list/list.tsx

```diff
@@ -82,6 +94,13 @@ const List = React.forwardRef((props: Props, ref?: React.Ref<MultiSelect<OptionV
         }
     }
 
+    let remainingText;
+    if (MAX_SELECTABLE_VALUES >= props.values.length) {
+        remainingText = 'Use ↑↓ to browse, ↵ to select. You can add {num, number} more {num, plural, one {person} other {people}}. ';
+    } else {
+        remainingText = 'Use ↑↓ to browse, ↵ to select. You can\'t add more than 7 people. Please <a>create a channel</a> to include more people. ';
+    }
```

Comment:
Changing just the default message doesn't work here. Let's use the FormattedMessage here directly. Also, it will avoid any weirdness in case the library complains about values being passed and not used.
```suggestion
    if (MAX_SELECTABLE_VALUES >= props.values.length) {
        remainingText = (
            <FormattedMessagedefineMessage
                id={'multiselect.numPeopleRemaining'}
                defaultMessage={'Use ↑↓ to browse, ↵ to select. You can add {num, number} more {num, plural, one {person} other {people}}. '}
                value={{
                    num: MAX_SELECTABLE_VALUES - props.values.length,
                }}
            />
        );
    } else {
        remainingText = (
            <FormattedMessagedefineMessage
                id={'multiselect.maxPeople'}
                defaultMessage={'Use ↑↓ to browse, ↵ to select. You can\'t add more than 7 people. Please <a>create a channel</a> to include more people.'}
                value={{
                    a: (chunks: React.ReactNode) => {
                        return (
                            <a
                                href='#'
                                onClick={(e) => handleCreateChannel(e)}
                            >{chunks}</a>
                        );
                    },
                }}
            />
        );
    }
```

Reply:
@larkox  I fixed it